services/ui-src/src/routes/McparReportPage/McparReportPage.tsx

```diff
@@ -87,12 +87,18 @@ const sx = {
     marginLeft: "3.5rem",
     h3: {
       paddingBottom: "0.75rem",
-      borderBottom: "1.5px solid var(--chakra-colors-palette-gray_lighter)",
       color: "palette.gray_medium",
       fontSize: "lg",
       fontWeight: "bold",
     },
     h4: {
+      paddingBottom: "0.75rem",
+      borderBottom: "1.5px solid var(--chakra-colors-palette-gray_lighter)",
+      color: "palette.gray_medium",
+      fontSize: "lg",
+      fontWeight: "bold",
+    },
+    h5: {
       fontSize: "lg",
       fontWeight: "bold",
     },
```

Comment:
`h3` should be #262626, which is the default, so deleting line 90 will get you there. `p` underneath it should also be that color, but it's being set earlier, so you'll have to explicitly set it here to "palette.base". 👍

Reply:
@scwambach Just need to delete line 90 and you're good to merge.